### PR TITLE
[CHORE] Reduce warnings

### LINKED ISSUE #1436: [CHORE] Resolve all project warnings

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios

Closes #1436

### CODE REVIEW

File: Rocket.Chat/Controllers/Auth/AuthViewController+LoginServices.swift [hunk 1]
```diff
@@ -0,0 +1,176 @@
+//
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Auth/AuthViewController+LoginServices.swift]: Following the ObjC categories style? 🤓

Comment (cardoso) [reply Rocket.Chat/Controllers/Auth/AuthViewController+LoginServices.swift]: What do you think? 🤔

Comment (rafaelks) [reply Rocket.Chat/Controllers/Auth/AuthViewController+LoginServices.swift]: @cardoso We're not using this approach on the other extensions... I like it, but then I think we would have to change the other files as well. 🙃

File: Rocket.Chat/API/Clients/MessagesClient.swift [hunk 2]
```diff
@@ -58,7 +58,6 @@ struct MessagesClient: APIClient {
         }, errored: { error in
             switch error {
             case .version:
-                // TODO: Remove SendMessage Fallback + old methods after Rocket.Chat 1.0
```
Comment (cardoso) [inline Rocket.Chat/API/Clients/MessagesClient.swift]: Let's just search for "case .version:" when we decide to remove fallbacks. That way we don't pollute with so many warnings.

**Comment by cardoso** [comment]
Ready for review @rafaelks

File: Rocket.Chat/API/Clients/MessagesClient.swift (hunk 2 above)
Comment (rafaelks) [reply Rocket.Chat/API/Clients/MessagesClient.swift]: 👍

**Pull request merged**